include/posix/posix_sched.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2018 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef __POSIX_SCHED_H__
+#define __POSIX_SCHED_H__
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* Cooperative scheduling policy */
+#ifndef SCHED_FIFO
+#define SCHED_FIFO 0
+#endif /* SCHED_FIFO */
+
+/* Priority based prempetive scheduling policy */
+#ifndef SCHED_RR
+#define SCHED_RR 1
```

Comment:
Currently  SCHED_RR and SCHED_FIFO are not used in our Zephyr Code so there is not possibility of getting these assigned to different value. Also in Linux too it is directly defined.
I wanted to avoid the complicated logic of multiple conditions here.